.github/workflows/reusable-e2e.yml

```diff
@@ -59,29 +60,24 @@ jobs:
         run: pnpm install --frozen-lockfile
       - name: Build Angular v${{ matrix.version }} E2E app
         run: pnpm build:source-map # with source map for bundle size job!
-      - name: Start Angular v${{ matrix.version }} E2E app server
-        run: pnpm start &
-      - name: Cypress run
-        uses: cypress-io/github-action@v6
-        with:
-          wait-on: 'http://localhost:4200'
-          working-directory: ${{ env.E2E_DIR }}
-          browser: chrome
+      # Temp for testing
+      #- name: Start Angular v${{ matrix.version }} E2E app server
+      #  run: pnpm start &
+      #- name: Cypress run
+      #  uses: cypress-io/github-action@v6
+      #  with:
+      #    wait-on: 'http://localhost:4200'
+      #    working-directory: ${{ env.E2E_DIR }}
+      #    browser: chrome
       - name: Analyze main bundle with source map explorer
-        run: |
-          pnpm run source-map-explorer-json &&
-          pnpm run bundle-pr-comment \
-            --git-ref '${{ github.event.pull_request.head.sha }}' \
-            --hidden-info '${{ env.BUNDLE_SIZE_COMMENT_ID_PREFIX }}${{ matrix.version }}'
+        run: pnpm run source-map-explorer-json
       - name: Upload main bundle analysis results
         uses: actions/upload-artifact@c7d193f32edcb7bfad88892161225aeda64e9392 # v4
         with:
           name: ${{ env.SOURCE_MAP_EXPLORER_ARTIFACT_NAME_PREFIX }}${{ matrix.version }}
           path: |
             ${{ env.E2E_APP_DIR }}/source-map-explorer.json
-            ${{ env.E2E_APP_DIR }}/bundle-size-pr-comment.md
-          # 👇 So we can download and compare from main
-          retention-days: ${{ github.ref_name == 'main' && 90 || 5 }}
+          retention-days: 5
```

Comment:
No need to use artifacts to store size, now stored on check runs, which assume last longer (can't find any place where it's said they're deleted)